src/tutorial/microbe_stage/MicrobeEngulfmentExplanation.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace Tutorial
+{
+    using System;
+    using Godot;
+    using Newtonsoft.Json;
+
+    public class MicrobeEngulfmentExplanation : TutorialPhase
+    {
+        [JsonProperty]
+        private Vector3? chunkPosition;
+
+        public MicrobeEngulfmentExplanation()
+        {
+            UsesPlayerPositionGuidance = true;
+        }
+
+        public override string ClosedByName { get; } = "MicrobeEngulfmentExplanation";
+
+        public override void ApplyGUIState(MicrobeTutorialGUI gui)
+        {
+            gui.EngulfmentExplanationVisible = ShownCurrently;
+        }
+
+        public override bool CheckEvent(TutorialState overallState, TutorialEventType eventType, EventArgs args,
+            object sender)
+        {
+            switch (eventType)
+            {
+                case TutorialEventType.MicrobeChunksNearPlayer:
+                {
+                    var data = (EntityPositionEventArgs)args;
+
+                    if (!HasBeenShown && data.EntityPosition.HasValue && CanTrigger && !overallState.TutorialActive())
```

Comment:
Should this verify the player is big enough to actually engulf the chunk? It would be a pretty bad experience to have a tutorial stuck on that you can't complete...

Reply:
And also make sure the player membrane is a type that allows engulfing.

Reply:
So all the size and can engulf, and cell wall, is already handled?

Reply:
I'll also call `CanEngulf` in there for good measure.